File: Selenium/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains

with webdriver.Firefox() as driver:

    actions = ActionChains(driver)

    # Open page
    driver.get("https://training-support.net/webelements/drag-drop")

    print("Page title:", driver.title)

    ball = driver.find_element(By.ID, "ball")
    dropzone1 = driver.find_element(By.ID, "dropzone1")
    dropzone2 = driver.find_element(By.ID, "dropzone2")

    actions.drag_and_drop(ball, dropzone1).perform()

    dz1_text = dropzone1.find_element(By.CLASS_NAME, "dropzone-text").text
    if dz1_text == "Dropped!":
        print("Ball successfully dropped in Dropzone 1")

    actions.drag_and_drop(ball, dropzone2).perform()

    dz2_text = dropzone2.find_element(By.CLASS_NAME, "dropzone-text").text
    if dz2_text == "Dropped!":
        print("Ball successfully dropped in Dropzone 2")

File: Selenium/Activities/Activity12.java
package examples;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity12 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        Actions actions = new Actions(driver);

        driver.get("https://training-support.net/webelements/drag-drop");

        System.out.println("Page title: " + driver.getTitle());

        WebElement ball = driver.findElement(By.id("ball"));
        WebElement dropzone1 = driver.findElement(By.id("dropzone1"));
        WebElement dropzone2 = driver.findElement(By.id("dropzone2"));

        actions.dragAndDrop(ball, dropzone1).perform();

        String dz1Text = dropzone1.findElement(By.className("dropzone-text")).getText();
        if (dz1Text.equals("Dropped!")) {
            System.out.println("Ball successfully dropped in Dropzone 1");
        }

        actions.dragAndDrop(ball, dropzone2).perform();

        String dz2Text = dropzone2.findElement(By.className("dropzone-text")).getText();
        if (dz2Text.equals("Dropped!")) {
            System.out.println("Ball successfully dropped in Dropzone 2");
        }

        driver.quit();
    }
}